src/components/AdminSettings/StunServers.vue

```diff
@@ -4,15 +4,10 @@
 -->
 
 <template>
-	<section id="stun_server" class="videocalls section">
-		<h2>
-			{{ t('spreed', 'STUN servers') }}
-		</h2>
-
-		<p class="settings-hint">
-			{{ t('spreed', 'A STUN server is used to determine the public IP address of participants behind a router.') }}
-		</p>
-
+	<NcSettingsSection id="stun_server"
+		:name="t('spreed', 'STUN servers')"
+		:description="t('spreed', 'A STUN server is used to determine the public IP address of participants behind a router.')"
+		doc-url="https://nextcloud-talk.readthedocs.io/en/latest/TURN/">
```

Comment:
stun is not turn?

Reply:
It is not, but on the TURN page (the "configuration/overview" page) there is a small paragraph describing what is STUN and when it is necessary.

> * Talk tries to establish a direct [peer-to-peer (P2P)](https://en.wikipedia.org/wiki/Peer-to-peer) connection, thus on connections beyond the local network (behind a [NAT](https://en.wikipedia.org/wiki/Network_address_translation) or router), clients do not only need to know each other's public IP, but the participants local IPs as well. Processing this, is the job of a [STUN](https://en.wikipedia.org/wiki/STUN) server. As there is one preconfigured for Nextcloud Talk that is operated by Nextcloud GmbH, for this case nothing else needs to be done.

Better to link to a Wiki article directly?

Reply:
I would say either it should be a useful installation/setup/configuration manual or we don't add a link